docs/receiving/receiving-with-ingest.mdx

```diff
@@ -239,7 +239,7 @@ can configure endpoints, view logs, inspect message payloads, etc.
 ![screenshot of the Ingest Dashboard showing the Destination/Endpoints tab for a Source named "demo"](../../static/img/ingest/destination-endpoints.png)
 
 Configuring endpoints allows you to forward messages received by Ingest over to
-the endpoints you choose using Svix Core.
+the endpoints you choose using Svix Sending.
```

Comment:
This is wrong, we don't use Svix Sending in Ingest, they are completely separate products.